include/fastdds/rtps/history/CacheChangePool.h

```diff
@@ -46,32 +46,24 @@ class CacheChangePool : public IChangePool
 
     /**
      * Constructor.
-     * @param initial_pool_size  Initial number of elements in the pool.
-     * @param max_pool_size      Maximum number of elements in the pool. If set to 0 the pool will keep reserving until something breaks.
-     * @param memory_policy      Memory management policy.
-     * @param f                  Functor to be called on all preallocated elements.
+     * @param config   Pool configuration.
```

Comment:
`CacheChange` ctors are not using the `config.payload_initial_size`, We should document it.